src/scss/themes/_full-bleed-image.scss

```diff
@@ -18,11 +18,6 @@
 			flex-direction: column;
 			justify-content: center;
 			padding: 20px;
-			min-height: 350px;
-		}
-
-		.o-topper__visual {
-			min-height: 350px;
```

Comment:
I think it's right to remove this one though. It's unhelpful unless we update the image to fill the visual area which I expect we don't want to do here — cropping images without editorial input is problematic